examples/typescript-backbone/js/app.ts

```diff
@@ -261,6 +285,28 @@ class TodoView extends Backbone.View {
 
 }
 
+// Todo Router
+// -----------
+
+class TodoRouter extends Backbone.Router {
+
+	routes = {
+		'*filter': 'setFilter'
+	};
+
+	constructor () {
+		super();
+		(<any>this)._bindRoutes();
+	}
+
+	setFilter (param: string) {
```

Comment:
Could you remove the space after the method name to follow the style used in the rest of the file? :lipstick: